components/brave_wallet/common/brave_wallet.mojom

```diff
@@ -1438,6 +1441,29 @@ struct AnkrAssetBalance {
   string price_usd;
 };
 
+struct NftAttribute {
+  string trait_type;
+  string value;
+};
+
+struct NftMetadata {
+  string name;
+  string description;
+  string image;
+  string image_data;
+  string external_url;
+  array<NftAttribute> attributes;
+  string background_color;
+  string animation_url;
+  string youtube_url;
+};
+
+struct NftIdentifier {
+  string chain_id;
+  string contract_address;
+  string token_id;
+};
+
```

Comment:
Let's also have coin here. `chain_id` alone in some cases might not be enough

Reply:
Including coin would also be nice to only require the one request for `GetNftMetadatas`, and one per account to `GetNftBalances` (if possible) 🙂